docs/getting_started.md

```diff
@@ -22,7 +22,7 @@ In this tutorial, we'll walk through the creation of some sample APBs.  We will
 
 Before getting started with APBs, we need to get your system set up to create them.
 
-First, make sure your system is properly running [OpenShift Origin](https://www.openshift.org/).  You should be running both the service catalog and Ansible Service Broker (ASB). Instructions can be found on the ASB [getting started doc](https://github.com/openshift/ansible-service-broker#getting-started-with-the-ansible-service-broker).
+First, make sure your system is properly running [OpenShift Origin](https://www.openshift.org/).  You should be running both the service catalog and Ansible Service Broker (ASB). Instructions can be found on the ASB repo. for Kubernetes[getting started doc](https://github.com/openshift/ansible-service-broker#getting-started-on-kubernetes) and OpenShift[getting started doc](https://github.com/openshift/ansible-service-broker#getting-started-on-openshift).
```

Comment:
I would add a space between Kubernetes and the link. ex 
`Kubernetes [getting started doc]`. Same with openshift and it's link